docs/api/tutorials/sdk/lineage_client.md

```diff
@@ -0,0 +1,365 @@
+
+```python
+from datahub.sdk import DataHubClient
+
+client = DataHubClient(server="<your_server>", token="<your_token>")
+
+# Getting column level lineage with source_urn and source_column
+results_with_source_column = client.lineage.get_lineage(
+    source_urn="urn:li:dataset:(urn:li:dataPlatform:snowflake,table_1,PROD)",
+    source_column="col1",
+    direction="downstream",
+    max_hops=2,
+)
+
+print(list(results_with_source_column))
+
+# Getting column level lineage with SchemaFieldUrn
+results_with_schema_field_urn = client.lineage.get_lineage(
+    source_urn="urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:snowflake,table_1,PROD),col1)",
+    direction="downstream",
+    max_hops=2
+)
+
+# this will return the same results as results_with_source_column
+print(list(results_with_schema_field_urn))
+```
+
+#### Interpreting Column Lineage Paths
+
+The `paths` field in the result represents traversal paths from the source column to downstream columns across one or more hops. Each path is a list of schema field URNs.
+
+```python
+results = [
+  LineageResult(
+    urn="urn:li:dataset:(urn:li:dataPlatform:snowflake,table_2,PROD)",
+    type="DATASET",
+    hops=1,
+    direction="downstream",
+    platform="snowflake",
+    name="table_2", # name of the entity 
+    paths=[
+      LineagePath(
+        urn="urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:snowflake,table_1,PROD),col1)",
+        column_name="col1", # name of the column
+        entity_name="table_1", # name of the entity that contains the column
+      ),
+      LineagePath(
+        urn="urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:snowflake,table_2,PROD),col4)",
+        column_name="col4", # name of the column
+        entity_name="table_2", # name of the entity that contains the column
+      )
+    ] # Only populated for column-level lineage
+  )
+]
+```
+
+For example, let's say we have the following lineage across three tables:
+
+![img_1.png](img_1.png)
+
+#### Example with `max_hops=1`
+
+```python
+>>> client.lineage.get_lineage(
+        source_urn="urn:li:dataset:(urn:li:dataPlatform:snowflake,table_1,PROD)",
+        source_column="col1",
+        direction="downstream",
+        max_hops=1
+    )
+```
+
+Returns:
+
+- `table_2` (hops=1)
+  - `table_1.col1 → table_2.col4`
+  - `table_1.col1 → table_2.col5`
+
+```python
+[
+    {
+        "urn": "...table_2...",
+        "hops": 1,
+        "paths": [
+            ["...table_1.col1", "...table_2.col4"],
+            ["...table_1.col1", "...table_2.col5"]
+        ]
+    }
+]
+```
+
+#### Example with `max_hops=2`
+
+```python
+>>> client.lineage.get_lineage(
+        source_urn="urn:li:dataset:(urn:li:dataPlatform:snowflake,table_1,PROD)",
+        source_column="col1",
+        direction="downstream",
+        max_hops=2
+    )
+```
+
+Returns:
+
+- `table_2` (hops=1)
+  - `col_1 → col_4`
+  - `col_1 → col_5`
+- `table_3` (hops=2)
+  - `col_1 → col_4 → col_7`
```

Comment:
probably need to reformat this but imo we should make it clear what the paths returns here